src/components/ExclusivityBanner/ExclusivityBanner.tsx

```diff
@@ -0,0 +1,71 @@
+import cn from "classnames";
+import { useContext, useEffect, useState } from "react";
+import Button from "../Button";
+import Icon from "../Icon";
+import styles from "./ExclusivityBanner.module.css";
+import ExclusivityContext from "./context";
+
+const STORAGE_KEY = "exclusivity_banner_dismissed";
+
+const ExclusivityBanner: React.FC = () => {
+  const { exclusiveFeature } = useContext(ExclusivityContext);
+  const [dismissed, setDismissed] = useState<boolean>(() => {
+    // Initialize from localStorage
+    if (typeof window !== "undefined") {
+      const stored = localStorage.getItem(STORAGE_KEY);
+      return stored === "true";
+    }
+    return false;
+  });
+
+  // Save to localStorage whenever dismissed state changes
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      localStorage.setItem(STORAGE_KEY, String(dismissed));
+    }
+  }, [dismissed]);
+
+  if (exclusiveFeature) {
+    return (
+      <div className={styles.exclusivityWrapper}>
+        <div
+          className={cn(styles.exclusivityBanner, {
+            [styles.visible]: !dismissed,
+          })}
+        >
+          <div className={styles.inner}>
+            <div className={styles.content}>
+              <Icon size="sm-md" name="rocketLaunch" />
+              <p>
+                {exclusiveFeature} is available only for Enterprise customers.{" "}
```

Comment:
Sorry, one thing I missed until just now: we updated the language, but it must have been after you worked on the first pass of implementation. Instead of `is available only for Enterprise customers.` it should be `is available only with Teleport Enterprise.`